lib/wallet/cardano-wallet.cabal

```
@@ -183,8 +183,10 @@ library
     Cardano.Wallet.DB.Store.Checkpoints.Migration
     Cardano.Wallet.DB.Store.Checkpoints.Store
     Cardano.Wallet.DB.Store.Delegations.Layer
-    Cardano.Wallet.DB.Store.Delegations.Migration
-    Cardano.Wallet.DB.Store.Delegations.Migration.Schema
+    Cardano.Wallet.DB.Store.Delegations.Migrations.Conway.Migration
+    Cardano.Wallet.DB.Store.Delegations.Migrations.Lean.Migration
+    Cardano.Wallet.DB.Store.Delegations.Migrations.Lean.Schema
+    Cardano.Wallet.DB.Store.Delegations.Migrations.Origin.Schema
```

```suggestion
    Cardano.Wallet.DB.Store.Delegations.Migrations.V5.Migration
    Cardano.Wallet.DB.Store.Delegations.Migrations.V3.Migration
    Cardano.Wallet.DB.Store.Delegations.Migrations.V3.Schema
    Cardano.Wallet.DB.Store.Delegations.Migrations.V2.Schema
```

Numbers don't require us to remember historical context. 🤔

---

I thought it was good to remember it ?

---

Perhaps

```
Cardano.Wallet.DB.Store.Delegations.Migrations.V5Conway.Migration
Cardano.Wallet.DB.Store.Delegations.Migrations.V3Lean.Migration
Cardano.Wallet.DB.Store.Delegations.Migrations.V3Lean.Schema
Cardano.Wallet.DB.Store.Delegations.Migrations.V2Origin.Schema
```

then? 🤔 This way, I feel that we have an easier time mapping version numbers to source code files.